### ISSUE TITLE
Successful to build but failed to run

### ISSUE THREAD

**Issue by 2dpdlja496or43iq** [yanghuan/CSharp.lua#427]
https://github.com/xiaoyuvax/bflata

Please have a look.

**Comment by yanghuan** [comment]
it is compie C# to native

**Comment by 2dpdlja496or43iq** [comment]
> it is compie C# to native

No. That project said it should build with any C# compilers (notice it has a .csproj?). The fact is dotnet 7.0 compiled it without any warnings but bflat generated full of warnings about some APIs not suitable to be AOT compiled.

**Comment by 2dpdlja496or43iq** [comment]
> bflata is an AOT (Ahead of Time) compiler that does not allow dynamically generated code in the project. Its warnings may indicate the usage of such dynamically generated code functionality in the project.

bflata (https://github.com/xiaoyuvax/bflata) is a build command generator. bflat (https://github.com/bflattened/bflat) is a compiler. bflat (https://github.com/bflattened/bflat) doesn't support .csproj like normal dotnet. bflata (https://github.com/xiaoyuvax/bflata) was created to generate bflat (https://github.com/bflattened/bflat) build command from .csproj. bflata (https://github.com/xiaoyuvax/bflata) is a generic code base that should be built with any C# compilers, including the compilers from .NET framework, Mono, dotnet, bflat (https://github.com/bflattened/bflat) and of course should be CSharp.lua too (that currently is not the case). I can't make it more clear. Please don't confuse bflata (https://github.com/xiaoyuvax/bflata) with bflat (https://github.com/bflattened/bflat).
